app/models/annotation/AnnotationService.scala

```diff
@@ -72,7 +72,7 @@ object AnnotationService
 
     def createTracing(dataSource: DataSource) = tracingType match {
       case TracingType.skeleton =>
-        dataSet.dataStore.saveSkeletonTracing(SkeletonTracingDefaults.createInstance.copy(dataSetName = dataSet.name, editPosition = dataSource.center))
+        dataSet.dataStore.saveSkeletonTracing(SkeletonTracingDefaults.createInstance.copy(dataSetName = dataSet.name, editPosition = dataSource.center, userBoundingBox = Some(BoundingBox(Point3D(510,510,510),5,5,5)))) //TODO: just for testing. remove before merge
```

Comment:
This can now be removed.

Reply:
done =) nice frontend implementation, works for me. Is there a way for the user to unset the bounding box? (make it null again)?

Reply:
It should be reset to null if the user enters an invalid bounding box (e.g. one with only 4 values or so) or if the user deletes everything from the bounding box input field.

Reply:
ah yes, I see it 👍